build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
```

Comment:
As an example, this PR references `rc3-23729` in test-runtime, but the packages just revved to `rc3-23802`